src/pr-info.ts

```diff
@@ -179,7 +179,6 @@ export async function queryPRInfo(prNumber: number) {
             query: GetPRInfo,
             variables: { pr_number: prNumber },
             fetchPolicy: "network-only",
-            fetchResults: true
```

Comment:
`fetchResults` was `true` by default and was removed https://github.com/apollographql/apollo-client/commit/eabf3e916bbd967e0f129f296c8681bef5ffed4f

Not sure it applied to `query()` at all?